File: Module-CP/Assignments/assignment14/Pass_by_value/q5.c
#include<stdio.h>
typedef struct salesManager{
    int id;
    char name[20];
    int salary;
    float commision;
    int incentive;
    int target;

}salesManager;
salesManager storeHR();
void display(salesManager);
int main(){
    salesManager h1,h2;

    h1=storeHR();
    h2=storeHR();
    
    printf("Details for HR-1:");
    display(h1);
    printf("Details for HR-2:");
    display(h2);
}
salesManager storeHR(){
    salesManager sm;

    printf("Enter ID:");
    scanf("%d",&sm.id);
    printf("Enter Name:");
    scanf("%s",sm.name);
    printf("Enter Salary:");
    scanf("%d",&sm.salary);
    printf("Enter Commision:");
    scanf("%f",&sm.commision);
    printf("Enter Incentive:");
    scanf("%d",&sm.incentive);
    printf("Enter Target:");
    scanf("%d",&sm.target);

    return sm;

}
void display(salesManager sm){
    printf("\nID:%d",sm.id);
    printf("\nName:%s",sm.name);
    printf("\nSalary:%d",sm.salary);
    printf("\nCommision:%f",sm.commision);
    printf("\nIncentive:%d",sm.incentive);
    printf("\nTarget:%d",sm.target);

}

File: Module-Core Java/Assignments/Java Assignments/Assignment 1/ByUsing_Constructors/q5.java
class HR{
    String name;
    int id;
    double salary;
    double comminsion;

     HR() {
        this.name="Pratik";
        this.id=1;
        this.salary=65761;
        this.comminsion=500;
    }

    public HR(String name, int id, double salary, double comminsion) {
        this.name = name;
        this.id = id;
        this.salary = salary;
        this.comminsion = comminsion;
    }

    void display(){
        System.out.println("Name: "+this.name);
        System.out.println("ID: "+this.id);
        System.out.println("Salary: "+this.salary);
        System.out.println("Commision: "+this.comminsion);
    }
}
public class q5{
    public static void main(String[] args) {
        HR e1=new HR();
        HR e2=new HR("Snehal",2,45000,400);

        e1.display();
        e2.display();
    }
}